src/components/SyncStatus/index.tsx

```diff
@@ -23,16 +23,16 @@ const SyncStatus = observer((props: Props) => {
                 <Icon
                     path={mdiSync}
                     spin={-2}
-                    color="var(--ifm-color-primary)"
+                    color="var(--sync-status-syncinc-color)"
```

Comment:
We were not consitent here, but i'm wondering wheter we should prefix all our custom css variables with `--tdev` s.t. the won't clash with user-projects?

Reply:
Good point, I think that would be useful – especially in light of our current efforts to get a clearer separation between teaching-dev and user projects. I'll change the newly introduced ones here and we can align the preexisting ones in a follow up.